tests/test_skvbc_slow_path.py

```diff
@@ -92,58 +95,63 @@ def setUp(self):
 
         Finally we check if a known K/V has been executed and readable.
         """
+        num_ops = 10
+        write_weight = 0.5
+
         bft_network.start_all_replicas()
-        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
 
         unstable_replicas = bft_network.all_replicas(without={0})
         crashed_replica = random.choice(unstable_replicas)
         bft_network.stop_replica(crashed_replica)
 
-        for _ in range(10):
-            await skvbc.write_known_kv()
+        _, slow_path_writes = await tracker.run_concurrent_ops(
+            num_ops=num_ops, write_weight=1)
 
         await bft_network.assert_slow_path_prevalent(as_of_seq_num=1)
 
         bft_network.start_replica(crashed_replica)
 
-        for _ in range(10):
-            key, val = await skvbc.write_known_kv()
+        await tracker.run_concurrent_ops(
+            num_ops=num_ops, write_weight=write_weight)
 
-        await bft_network.assert_fast_path_prevalent(nb_slow_paths_so_far=10)
+        await trio.sleep(5)
+        await bft_network.assert_fast_path_prevalent(nb_slow_paths_so_far=slow_path_writes)
 
-        await skvbc.assert_kv_write_executed(key, val)
 
     @with_trio
     @with_bft_network(start_replica_cmd)
-    async def test_slow_path_view_change(self, bft_network):
+    @verify_linearizability
+    async def test_slow_path_view_change(self, bft_network, tracker):
         """
         This test validates the BFT engine's transition to the slow path
         when the primary goes down. This effectively triggers a view change in the slow path.
 
-        First we write a batch of known K/V entries.
+        First we write a batch of K/V entries and track them using the tracker from the decorator.
 
         We check those entries have been processed via the fast commit path.
 
-        We stop the primary and send a batch of requests, triggering slow path & view change.
+        We stop the primary and send a indefinite batch of tracked read & write requests,
+        triggering slow path & view change.
 
         We bring the primary back up.
 
         We make sure the second batch of requests have been processed via the slow path.
         """
 
+        num_ops = 10
+        write_weight = 0.5
         bft_network.start_all_replicas()
-        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
 
-        for _ in range(10):
-            await skvbc.write_known_kv()
+        await tracker.run_concurrent_ops(
+            num_ops=num_ops, write_weight=1)
 
         await bft_network.assert_fast_path_prevalent()
 
         bft_network.stop_replica(0)
 
         with trio.move_on_after(seconds=5):
             async with trio.open_nursery() as nursery:
-                nursery.start_soon(skvbc.send_indefinite_write_requests)
+                nursery.start_soon(tracker.send_indefinite_tracked_ops, write_weight)
 
         bft_network.start_replica(0)
 
```

Comment:
Next line should say:
`await bft_network.wait_for_slow_path_to_be_prevalent(as_of_seq_num=fast_path_writes)`